### ISSUE TITLE
Add Image scanning and signing to the CI/CD repo

### ISSUE THREAD

**Issue by springdo** [rht-labs/labs-ci-cd#182]
Using Andy Blocks examples as a starting point
https://github.com/redhat-cop/openshift-image-signing-scanning
or https://github.com/sabre1041/image-scanning-signing-service for potentially more up to date informantion
ACs
- [ ] Create the image-signing-scanning project as part of CI
- [ ] Create a ref example in the `todolist-fe` pipeline

@Tompage1994  - we can pair on this at some point if you want?

**Comment by Tompage1994** [comment]
Yeah I had a quick look at this this morning. Looks a little complex but looks good as well. I know some the previous customer wanted some stuff around this!
